diff --git a/cmake/sip_configure.py b/cmake/sip_configure.py
index 5686b00..bcfd73f 100644
--- a/cmake/sip_configure.py
+++ b/cmake/sip_configure.py
@@ -94,7 +94,7 @@ default_platform_lib_function = sipconfig.SIPModuleMakefile.platform_lib
 
 
 def custom_platform_lib_function(self, clib, framework=0):
-    if os.path.isabs(clib):
+    if os.path.isabs(clib) or clib.startswith('-l'):
         return clib
     return default_platform_lib_function(self, clib, framework)
 sipconfig.SIPModuleMakefile.platform_lib = custom_platform_lib_function
